packages/cli/src/command.ts

```diff
@@ -1,8 +1,34 @@
 import {Command as OclifCommand} from '@oclif/command'
 import Enquirer = require('enquirer')
+import chalk = require('chalk')
+
+export type LogOptions = {
+  pad?: boolean
+  indent?: boolean
+}
 
 abstract class Command extends OclifCommand {
   protected enquirer = new Enquirer()
+  protected themeColor = '6700AB'
+  protected indent = false
+
+  logTheme(str: string, options?: LogOptions) {
+    this.log(chalk.bgWhite.hex(this.themeColor).bold(str), options)
```

Comment:
Can we reuse the `log` util for these? I've opened #194 for adding a utils package, but importing from `server` is fine for now.

Reply:
Thanks! Better to make a step towards the right solution than no step at all :)